sheetstorm/settings/base.py

```diff
@@ -268,3 +268,6 @@
 ]
 
 CRISPY_TEMPLATE_PACK = 'bootstrap4'
+
+# List of valid email domains which are allowed in https://s.codepoets.it
```

Comment:
Maybe not in `https://s.codepoets.it` because you are pointing on domain which should use it. I think it better comment for this might be: 
```
# List of valid email domains which are allowed in SheetStorm
```

Reply:
Done